### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by juancarlospaco**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ See something you think belongs here? Feel free to fork and send a PR!
 - [The Nim forum](http://forum.nim-lang.org/)
 ### Libraries
 - [html2karax](https://github.com/nim-lang-cn/html2karax) - Converts static html to Karax in Nim
+- [faster-than-csv](https://github.com/juancarlospaco/faster-than-csv#csv2karax) - Convert CSV to Karax table
```
Comment (Araq) [inline README.md:26]: faster-than-csv is nice but why the Python dependency?

Comment (juancarlospaco) [reply README.md:26]: It is for Python.

Comment (Araq) [reply README.md:26]: Then it's not for Karax, is it?

Comment (juancarlospaco) [reply README.md:26]: The same way some people work doing C++ modules for Python,
some of us do Nim modules for Python.
Python cant do Frontend thats why.

Comment (Araq) [reply README.md:26]: Meh I still don't get it but how about I simply accept it and move on...

Comment (juancarlospaco) [reply README.md:26]: I understand is a "gray" area, is Ok if not merged too, 
I just think Karax needs more popularity, wherever it is used. Thanks. 🙂

**Pull request merged**